api/v1alpha1/validatedelete_test.go

```diff
@@ -0,0 +1,34 @@
+package v1alpha1
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("ValidateDelete methods", func() {
+	It("Should return nil", func() {
+		release := &Release{
+			TypeMeta:   metav1.TypeMeta{},
+			ObjectMeta: metav1.ObjectMeta{},
+		}
+		Expect(release.ValidateDelete()).To(BeNil())
+	})
+
+	It("Should return nil", func() {
+		releaseplan := &Release{
+			TypeMeta:   metav1.TypeMeta{},
+			ObjectMeta: metav1.ObjectMeta{},
+		}
+		Expect(releaseplan.ValidateDelete()).To(BeNil())
+	})
+
+	It("Should returns nil", func() {
```

Comment:
it may be nice to include the CRD name in the It line so it is easy to narrow down which test failed if one ever does

Reply:
You mean the string "releaseplan"? 
Also, I guess the tense for "returns" should probably not be plural.

Reply:
So I moved these tests into the webhook tests for each struct, and deleted the `api/v1alpha1/validatedelete_test.go`.
Also fixed them to actually test the `ValidateDelete()` for each struct.